tasks/test_qdr.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: "[Test] Check that qdr container is running."
+  shell: |
+    podman ps | grep {{ qdr_container_name }}
```

Comment:
if the container is managed by tripleo, you should use systemdctl to check if the systemd is active and running (which manages the container). Doing grep will certainly leads to failures later; if a container has similar name and such...

Reply:
That makes sense,  but we still need to check if the container is running when we're using docker on OSP13.

I can add in a check here, and select the proper/systemctl check when podman is used, and use this grep when OSP13/docker are used.

This is part of issue https://github.com/infrawatch/functional-tests/issues/7, when there's a container_bin var to check and use for selection